unfccc/errors.go

```diff
@@ -5,5 +5,5 @@ import (
 )
 
 var (
-	ErrNoMatchingRegion error = errors.New("No region matched that region code")
+	ErrNoMatchingRegion error = errors.New("no region matched that region code")
```

Comment:
Go is opinionated about errors and they should be lower case. Fixes a warning from golint.